### ISSUE TITLE
Does this neural network work for other customized dataset?

### ISSUE THREAD

**Issue by Fei-Xia-oe** [soltanianzadeh/STNeuroNet#13]
I want to apply this neural network directly to my acquired data for segmentation but I don't have much data for training. Will this method for my case? Thanks!

**Comment by soltanianzadeh** [comment]
It is not easy to comment on this. It depends on your data acquisition settings. For example, what brain area are you imaging? What is the pixel size or resolution of you data? What is the frame rate? or what is the calcium sensor? 

You can definitely try passing your data through the provided trained network (on the ABO dataset) after the pre-processing steps described in the paper (temporal binning, filtering and normalization) and visually evaluate the results. If not satisfactory, you can fine-tune the trained network on your (limited) training data to customize it for your acquisition settings. Alternatively, you could try unsupervised domain-adaptation techniques that do not require labelled data from the target domain, which is your dataset.
